### DISCUSSION TITLE
Connecting....

### DISCUSSION THREAD

**Discussion by Franjo-web** [rustdesk/rustdesk#9145] [General]
Hello,
I have a self-hosted server, and I was able to connect everything. About one month ago, one of the connections stopped, and it's my PC at home. Other connections are in the company, and I can connect to them from the company. From my home PC, I can connect to my company PCs.
![image](https://github.com/user-attachments/assets/e16958d8-d3fd-4d1b-a915-2d76002c126b)

Everything looks ok, and when I try to connect, It says "Connecting.." forever and then timeout and again connecting.
![image](https://github.com/user-attachments/assets/f19f6a3f-fec8-4863-b228-6d154ccbe8cf)

Any suggestions?

**Comment by wxdjf0212** [comment]
pls use version 1.2.3-2，it will be ok
I also encountered the same problem because I upgraded to a new version

**Reply by wxdjf0212** [reply]
1.2.3-2 is client version

**Reply by Franjo-web** [reply]
So the fix for this is to downgrade to 1.2.3-2 version?

**Reply by Franjo-web** [reply]
No, it didn't solve the problem, and also with that version client is only restarting xD.

**Comment by Franjo-web** [comment]
I'ts very strange that I can connect from home to this work PC and I cant from this work PC connect to home xD

**Comment by rustdesk** [comment]
#763, go to [discord](https://discord.gg/nDceKgxnkV) please, usually nobody will help you with self-hosting here.
